specifications/base.protocol.md

```diff
@@ -66,7 +66,7 @@ authorization request parameter set to a list of space-delimited scopes the `tok
 `bearer_access_scope` parameter is present, the `token` claim should not be included.
 
 <aside class="note">
-A non-normative OpenAPI spec of an [=STS=] implementing client credentials flow is provided [here](specifications/identity-trust-sts-api.yaml)
+A non-normative OpenAPI spec of an [=STS=] implementing client credentials flow is provided [here](identity-trust-sts-api.yaml)
```

Comment:
Doesn't the yaml file contain the exact same content as the newly introduced http .txt? We should probably remove this.